### PR TITLE
#362 : Rotated Text Implementation

### PR DESCRIPTION
**Pull request by memoyil**
Rotated Text Implementation

### CODE REVIEW

**Comment by yegor256** [comment]
@memoyil can you please update our compatibility table in README, to show which features from Apach or Guava we're copying.

**Comment by memoyil** [comment]
@yegor256 Ok. I will apply it.

**Comment by memoyil** [comment]
@yegor256  Done.

File: src/main/java/org/cactoos/text/RotatedText.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    public String asString() throws IOException {
+        String text = this.origin.asString();
+        final int length = text.length();
+        if (length != 0 && this.move != 0 && this.move % length != 0) {
+            final StringBuilder builder = new StringBuilder(length);
+            int offset = -(this.move % length);
+            if (offset < 0) {
+                offset = text.length() + offset;
+            }
+            text = builder.append(text.substring(offset)).append(text
```
Comment (yegor256) [inline src/main/java/org/cactoos/text/RotatedText.java]: @memoyil see http://www.yegor256.com/2014/10/23/paired-brackets-notation.html

**Comment by yegor256** [comment]
@memoyil one small comment above

**Comment by memoyil** [comment]
@yegor256 Done.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**